taroapp/cmd/config.py

```diff
@@ -2,15 +2,34 @@
 Commands related to configuration.
 Show: shows path and content of the config file
 """
+from shutil import copy
+from pathlib import Path
 
-from taro import cfgfile
+
+from taro import cfgfile, paths
 from taroapp import argsconfig, cli
 
 
-def run(args):
-    config_path = argsconfig.get_config_file_path(args)
 
+def run(args):
     if args.config_action == cli.ACTION_CONFIG_SHOW:
+        config_path = argsconfig.get_config_file_path(args)
         cfgfile.print_config(config_path)
+    if args.config_action == cli.ACTION_CONFIG_CREATE:
+        create(args)
     if args.config_action == cli.ACTION_CONFIG_PATH:
+        config_path = argsconfig.get_config_file_path(args)
         print(config_path)
+
+
+def create(args):
+    current_dir_path = Path.cwd() / 'taro.yaml'
```

Comment:
**taro.yaml** is already defined in **paths** module as variable **DEFAULT_CONFIG_FILE**. Please use this variable.